src/CrossReferences.jl

```diff
@@ -53,11 +53,12 @@ function basicxref(link::Markdown.Link, meta, page, doc)
     elseif isa(link.text, Vector)
         # No `name` was provided, since given a `@ref`, so slugify the `.text` instead.
         text = strip(sprint(Markdown.plain, Markdown.Paragraph(link.text)))
-        if occursin(r"#[0-9]+", text)
-            issue_xref(link, lstrip(text, '#'), meta, page, doc)
-        else
+        refissue = match(r"#[0-9]+", text)
+        if refissue === nothing
             name = Utilities.slugify(text)
             namedxref(link, name, meta, page, doc)
+        else
+            issue_xref(link, refissue[1], meta, page, doc)
```

Comment:
I don't think this is correct -- there is no capture group in regex:

```julia
julia> x = match(r"#[0-9]+", "#124")
RegexMatch("#124")

julia> x[1]
ERROR: BoundsError: attempt to access 0-element Array{Union{Nothing, SubString{String}},1} at index [1]
```

Either way, as far as I can tell, this change is strictly non-functional? In which case I would just remove it -- it's debatable whether this actually improves readability, so I would rather err on the side of not making unnecessary non-functional changes.

Reply:
I'm not sure where the grouping went in this commit. I agree that this change isn't important.